lib/extras/ifdef.dart

```diff
@@ -0,0 +1,4 @@
+class IfDef {
```

Comment:
I'm not sure about the reason for using the switch for animation. Is it for test purposes?

Reply:
It was for testing, so now it is removed.